shell/server/pre-meteor.js

```diff
@@ -471,6 +490,12 @@ Meteor.startup(() => {
         return;
       }
 
+      if (id.match(/^selftest/)) {
```

Comment:
I always use `Random.hexString(20)` in the "client" code for this, so I'm +1 on me changing this to be a different length, too.

I'd prefer the length-based way to differentiate the selftest hosts, but I make it use a hyphen if you prefer that.